services/server/src/server/services/utils/database-util.ts

```diff
@@ -607,7 +575,7 @@ export async function getSourcifyMatchAddressesByChainAndMatch(
   page: number,
   paginationSize: number,
   descending: boolean = false,
-) {
+): Promise<QueryResult<{ address: string }>> {
   let queryWhere = "";
   switch (match) {
     case "full_match": {
```

Comment:
(not related to this line, just couldn't pick the right one)

 The `updateContractDeployment` function would also need a return type.

Reply:
it actually doesn't need a specific return type, I removed the `RETURN *`

Reply:
Okay great :+1: